src/resources/styles.ts

```diff
@@ -110,12 +110,16 @@ export const haStyle = css`
     color: var(--app-header-text-color, white);
   }
 
-  app-toolbar [main-title] {
-    margin-left: 20px;
+  app-toolbar {
+    height: var(--header-height);
   }
 
-  ha-menu-button {
-    margin-left: 4px;
+  app-header div[sticky] {
+    height: 48px;
```

Comment:
48 or header-height ?

Reply:
Sticky has always been 48, but when we changed to the variable and made app header 64px again it didn't get changed. It is correct at 48 now.